.github/workflows/identity-and-trust.yml

```diff
@@ -0,0 +1,83 @@
+name: "[Identity & Trust] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 9:30AM UTC (10:30AM BST), Mondays to Fridays
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: "0 8 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        # Generated by running the following query in the service-catalogue database:
+        # SELECT short_repo_name FROM view_repo_ownership WHERE (github_team_name = 'Identity' OR github_team_name = 'Discussion' OR github_team_name = 'Transparency & Consent') AND NOT archived
+        run: |
+          REPOS=(
+            cmp-market-research
+            consent-management-platform
+            baton
+            transparency-consent-docs
+            baton-lambda-template
+            discussion-api
+            discussion-platform
+            grl
+            hmac-headers
+            identity
+            identity-account-deletion
+            identity-admin
+            salesforce-message-handler
+            hackday-gustodon-bot
```

Comment:
Do we mean to include this repository? I suspect it might be a candidate for archiving.

Reply:
Removed! (not archived yet though)